luaui/Widgets/gui_pregame_build.lua

```diff
@@ -158,6 +159,11 @@ function widget:Initialize()
 		end
 	end
 
+	local metalSpots = WG["resource_spot_finder"].metalSpotsList
+	if not metalSpots or (#metalSpots > 0 and #metalSpots <= 2) then
```

Comment:
I hate this, maps should have a way to report themselves as metal maps

Reply:
What about existing maps?

Reply:
Assuming we knew how to assign metadata we should make them report themselves as well.

Reply:
If that metadata could be assigned in rowy or something that would be great. This is definitely a yucky hack